From 1e3e696928ae8030dc568df654b303b3a37e1c94 Mon Sep 17 00:00:00 2001
From: "Ethan D. Twardy" <ethan.twardy@gmail.com>
Date: Sat, 16 Mar 2024 16:57:14 -0500
Subject: [PATCH 2/3] Fix .gitconfig and email addresses

 * Don't gpgsign commits with my personal key
 * Don't use my git-sendemail configuration
 * Don't verify SSL certificates
 * Replace all references to my personal email with my work email
---
 home/.bash_aliases |  2 +-
 home/.gitconfig    | 12 +++---------
 home/.makepkg.conf |  2 +-
 3 files changed, 5 insertions(+), 11 deletions(-)

diff --git a/home/.bash_aliases b/home/.bash_aliases
index cf352f4..3c3d750 100644
--- a/home/.bash_aliases
+++ b/home/.bash_aliases
@@ -27,7 +27,7 @@ alias rm='rm -i'
 alias svn='svn --no-auth-cache'
 alias tmux="tmux -f $HOME/.tmux.conf"
 
-export EMAIL=ethan.twardy@gmail.com
+export EMAIL=ethan.twardy@plexus.com
 export FULLNAME="Ethan D. Twardy"
 
 # Used by various debian package maintainer tools
diff --git a/home/.gitconfig b/home/.gitconfig
index 8ff7a71..08c9564 100644
--- a/home/.gitconfig
+++ b/home/.gitconfig
@@ -1,7 +1,6 @@
 [user]
-	email = ethan.twardy@gmail.com
+	email = ethan.twardy@plexus.com
 	name = Ethan D. Twardy
-	signingkey = 0FF164523340E827
 [core]
 	editor = /usr/bin/vi
 	autocrlf = false
@@ -9,6 +8,8 @@
 	defaultBranch = main
 [pull]
 	rebase = false
+[http]
+  sslVerify = false
 [grep]
 	lineNumber = true
 [filter "lfs"]
@@ -16,12 +17,5 @@
 	smudge = git-lfs smudge -- %f
 	process = git-lfs filter-process
 	required = true
-[commit]
-	gpgsign = true
-[sendemail]
-	smtpServer = smtp.gmail.com
-	smtpServerPort = 587
-	smtpEncryption = tls
-	smtpUser = ethan.twardy@gmail.com
 [color]
 	ui = auto
diff --git a/home/.makepkg.conf b/home/.makepkg.conf
index 639ba68..af8819a 100644
--- a/home/.makepkg.conf
+++ b/home/.makepkg.conf
@@ -1 +1 @@
-PACKAGER="Ethan D. Twardy <ethan.twardy@gmail.com>"
+PACKAGER="Ethan D. Twardy <ethan.twardy@plexus.com>"
-- 
2.44.0

